vufs/src/main/java/com/stnetix/ariaddna/vufs/BusinessObjects/Metatable.java

```diff
@@ -0,0 +1,55 @@
+package com.stnetix.ariaddna.vufs.BusinessObjects;
+
+import com.stnetix.ariaddna.commonutils.mavenutil.MavenUtil;
+
+import java.util.Set;
+import java.util.concurrent.CopyOnWriteArraySet;
+
+/**
+ * Metatable it's kernel structure of VUFS and implementing as acyclic graph.
+ */
+public class Metatable {
+
+    private String version;
+    private MetatableType type;
+    private Set<Metafile> metafileSet;
+    private Long lastUpdateTimestamp;
+
+    public Metatable(MetatableType type) {
+        this.type = type;
+        version = MavenUtil.getCurrentVersion();
+        metafileSet = new CopyOnWriteArraySet<>();
+    }
+
+    public boolean addMetafileUUid(Metafile metafile) {
+        if (metafileSet.add(metafile)) {
+            lastUpdateTimestamp = System.currentTimeMillis();
```

Comment:
Don't use "System.currentTimeMillis()" because it make tests is more difficult. 
ariADDna has a class com.stnetix.ariaddna.commonutils.datetime.DateTime. 
If need add additional functions to this class.